src/Microsoft.ML.LightGBM/LightGbmMulticlassTrainer.cs

```diff
@@ -45,7 +45,7 @@ internal LightGbmMulticlassTrainer(IHostEnvironment env, Options options)
         /// Initializes a new instance of <see cref="LightGbmMulticlassTrainer"/>
         /// </summary>
         /// <param name="env">The private instance of <see cref="IHostEnvironment"/>.</param>
-        /// <param name="labelColumn">The name of The label column.</param>
+        /// <param name="labelColumn">The name of the label column.</param>
         /// <param name="featureColumn">The name of the feature column.</param>
         /// <param name="weights">The name for the column containing the initial weight.</param>
```

Comment:
>The name for the column containing the initial weight. [](start = 34, length = 54)

`/// <param name="weightsColumnName">The name of the optional weights column.</param>`